sections/testingandquality/refactoring.md

```diff
@@ -0,0 +1,27 @@
+# Refactoring
+
+<br/><br/>
+
+### Motivation for Refactoring
```

Comment:
@TheHollidayInn Should be "One Paragraph Explainer" (consistencty with other bullets)